quarto/src/Repository/GameRepository.php

```diff
@@ -0,0 +1,23 @@
+<?php
+namespace App\Repository;
+use Doctrine\ORM\EntityRepository;
+use Doctrine\ORM\EntityManagerInterface;
+use App\Entity\Game;
+class GameRepository extends EntityRepository {
+  private $em;
+  public function __construct(EntityManagerInterface $em) {
+    $this->em = $em;
+  }
+  public function findGameById(string $id) : Game {
+    return $this->em->find('App:Game', $id);
```

Comment:
It's strange to extends EntityRepository and use entityManager directly. EntityRepository already have a "findById" method called "find". So please use it instead

https://www.doctrine-project.org/api/orm/latest/Doctrine/ORM/EntityRepository.html